### ISSUE TITLE
problem with enums rendering on ui

### ISSUE THREAD

**Issue by mherb63** [swagger-api/swagger-codegen#1514]
We have the following defined in our yaml:

![pastedgraphic-8](https://cloud.githubusercontent.com/assets/1205814/10983358/1595c8dc-83e0-11e5-85fd-8e973eaa15bb.png)

But, when its rendered on the UI it shows up as follows:  notice the { and ] characters

![pastedgraphic-9](https://cloud.githubusercontent.com/assets/1205814/10983375/30fe1fd4-83e0-11e5-8b90-e3bf1fc09d7a.png)

**Comment by webron** [comment]
Are you talking about swagger-ui?

**Comment by mherb63** [comment]
yes, sorry I should have mentioned that the code was generated from swaggerhub.com  I generated a Spring MVC server project.

**Comment by webron** [comment]
To be sure and pin-point the problem, can you describe the steps you've taken to see the problem? I want to see how to reproduce it, and point you to the right project if it's not this one.

**Comment by mherb63** [comment]
here's the relavant yaml:

<pre>
  /SCA_ETH_FDFr_EC/{id}:
    patch:
      tags:
      - "SCA_ETH_FDFr_EC"
      description: "Activate or Deactivate an existing Ethernet Flow Domain Fragment\
        \ (SCA_ETH_FDFr_EC) identified with {id}. Use state variable for action. "
      deprecated: false
      parameters:
      - name: "id"
        in: "path"
        required: true
        description: "id of Ethernet Flow Domain Fragment"
        type: "string"
      - name: "state"
        in: "query"
        description: "state of SCA_ETH_FDFr_EC"
        type: "string"
        enum:
        - "active"
        - "inactive"
        - "suspended"
        - "terminated"
        required: true
      responses:
        200:
          description: "Ethernet Flow Domain Fragment EC"
          schema:
            $ref: "#/definitions/SCA_ETH_FDFr_EC"
        400:
          description: "Invalid SCA_ETH_FDFr PATCH"
</pre>

here's what the generated controller method looks like:  does that look correct?

    @ApiOperation(value = "", notes = "Activate or Deactivate an existing Ethernet Flow Domain Fragment (SCA_ETH_FDFr_EC) identified with {id}. Use state variable for action.", response = SCAETHFDFrEC.class)
    @ApiResponses(value = { @ApiResponse(code = 200, message = "Ethernet Flow Domain Fragment EC") })
    @RequestMapping(value = "/{id}", produces = { "application/json" }, method = RequestMethod.PATCH)
    public ResponseEntity<SCAETHFDFrEC> sCAETHFDFrECIdPatch(@ApiParam(value = "id of Ethernet Flow Domain Fragment", required = true) @PathVariable("id") String id,
            @ApiParam(value = "state of SCA_ETH_FDFr_EC", required = true, allowableValues = "{values=[active, inactive, suspended, terminated]}") @RequestParam(value = "state", required = true) String state) throws NotFoundException, ApiException {
        return new ResponseEntity<SCAETHFDFrEC>(this.mefScaService.patchFlowDomainFragment(id, state), HttpStatus.OK);
    }

**Comment by fehguy** [comment]
That looks like a bug in the templates.
